src/parser.rs

```diff
@@ -376,9 +381,19 @@ impl Parser {
             self.record_or_closure()
         } else if self.is_lparen() {
             self.lparen();
-            let output = self.expression();
-            self.rparen();
-            output
+            if self.is_rparen() {
+                self.compiler.blocks.push(Block::new(vec![]));
```

Comment:
We can just disallow `()` because it's useless, you can use `null`. The error message could say "use null instead of ()". Adding a block here is unnecessary complexity and inconsistent with the `self.expression()` below because blocks also allow statements.